src/game_state/game.py

```diff
@@ -9,16 +17,14 @@
 
 
 class Game:
-
-    snake_speed = 15
-
-    # Window size
-    window_x = 720
-    window_y = 480
+    """Class representing the Game entity."""
 
     def __init__(self):
         pygame.init()
         pygame.display.set_caption("Snake")
+        self.snake_speed = 15
+        self.window_x = 720
+        self.window_y = 480
```

Comment:
These should be at least optional parameters. In the future, we could make the window size configurable or let the user select the difficulty level.

Reply:
Ack